### PR TITLE
[sil.tea.dict] Initial creation

### PR DESCRIPTION
**Pull request by darcywong00**
Submitting lexical-model on behalf of @lublinkhof for Temiar

To accompany keymanapp/keyboards#2202

### CODE REVIEW

**Comment by darcywong00** [comment]
@lublinkhof, there seem to be some extra characters in wordlist.tsv.

Do you know if these are correct?
2 instances of <kbd>ˈ</kbd> which is U+02C8 MODIFIER LETTER VERTICAL LINE (not the one you use for glottal stop)
umˈayèny	1
jemaˈat	1

**Comment by darcywong00** [comment]
Per @lublinkhof, those two should be glottal stops.

File: release/sil/sil.tea.dict/source/sil.tea.dict.model.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  This is a minimal lexical model source that uses a tab delimited wordlist.
+  See documentation online at https://help.keyman.com/developer/ for
+  additional parameters.
+*/
+
+const source: LexicalModelSource = {
+  format: 'trie-1.0',
+  wordBreaker: 'default',
+  sources: ['wordlist.tsv'],
+};
```
Comment (darcywong00) [inline release/sil/sil.tea.dict/source/sil.tea.dict.model.ts:13]: I've tried setting `languageUsesCasing: true` but I'm not seeing a difference...

**Review by DavidLRowe — APPROVED** [review]

**Comment by DavidLRowe** [comment]
@darcywong00 Sorry I didn't see this sooner. Thanks for pinging me with the review request.

**Comment by darcywong00** [comment]
No worries. @lublinkhof - Keyman should install this when your users install a Temiar keyboard (language tag "tea")

**Pull request merged**